src/App.css

```diff
@@ -2,3 +2,338 @@
   margin: 0;
   padding: 0;
 }
+
+body {
+  font-family: 'Times New Roman', Times, serif;
+  background-color: #f3f4f8;
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
+  background-color: #fff;
+  margin-top: 50px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+.nave-bar {
+  max-width: 1200px;
+  display: flex;
+  padding: 20px;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
```

Comment:
- Great Job For having this beautiful UI 👍👍 however you have to stick it to the given [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design as it is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) 👌
### here is your design
![Screenshot from 2023-03-03 15-53-11](https://user-images.githubusercontent.com/52010789/222728487-f8c76e66-3a7d-4c58-922e-4eb71db3a44d.png)
![Screenshot from 2023-03-03 15-53-21](https://user-images.githubusercontent.com/52010789/222728491-c0658466-6e3d-4410-b0d1-80ed06b2b927.png)


### here is [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design
![Screenshot 2023-03-03 at 12 02 18 PM](https://user-images.githubusercontent.com/52010789/222679538-da064e37-5a97-4180-b20b-39edf84e5dc3.png)